src/modules/Dropdown/Dropdown.js

```diff
@@ -335,6 +340,25 @@ export default class Dropdown extends Component {
     }
   }
 
+  removeItemOnBackspace = (e) => {
+    debug('removeItemOnBackspace()')
+    debug(keyboardKey.getName(e))
+    if (keyboardKey.getCode(e) !== keyboardKey.Backspace) return
+
+    const { multiple, search } = this.props
+    const { searchQuery, value } = this.state
+
+    if (searchQuery || !search || !multiple || !value) return
```

Comment:
Found a little bug here.  When there are not items left and you hit backspace, this method still calls onChange and setValue.  This is because `!value` here is still true since it is an empty array.

Being that multiple dropdowns have arrays for value we should use `_.isEmpty(value)` here instead.  This will be true if empty, false if there are any items in the array.